docs/concepts/advanced/extensions.md

```diff
@@ -15,10 +15,10 @@ Extensions can be set by either as an import string or by passing in a class:
 ```python
 # settings.py
 
-class MyExtension(ComponentsExtension):
+class MyExtension(ComponentExtension):
     name = "my_extension"
 
-    class ExtensionClass(BaseExtensionClass):
```

Comment:
In the docs we ask people to subclass from `BaseExtensionClass` in order to define the "extension class" which would add some nested class to components, e.g.:

```py
class Table(Component):
    class MyExtension:
        ...
```

But `BaseExtensionClass` wasn't publically exposed. Which made me think that maybe it could be easier to access and remember that base class as `ComponentExtension.ExtensionClass`.